### ISSUE TITLE
Badge builder & new home (WIP)

### ISSUE THREAD

**Issue by amio** [amio/badgen-service#192]
I'm making a [badge-builder](https://badgen.net/builder) to help select/create/preview badge, also it's meant to be the new homepage of Badgen. It's still work in progress, but it shows the general ideas now.

What do you think about it? Is there any advice for this?

**Comment by styfle** [comment]
I just visited the website on my phone and I don’t really see what it’s supposed to do 🤷‍♂️

**Comment by amio** [comment]
Totally not work on mobile (yet) 😄

**Comment by evelynhathaway** [comment]
I love the work you've done on it! I got excited when I saw it when testing the site locally a few days ago.

> What do you think about it? Is there any advice for this?

I think continuing the supportive helper text underneath the input field to be more aware of the URI structure would be great. Minecraft's recent rework of their command system employs a good design pattern [worth taking a look at](https://youtu.be/svuDA9sM6_o?t=145), as it is slightly more analogous to what would work well compared to code completion in a text editor.
